european-journal-of-endocrinology.csl

```diff
@@ -0,0 +1,143 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="symbol" et-al-use-last="true" page-range-format="expanded" initialize-with-hyphen="false" default-locale="en-GB">
+  <info>
+    <title>European Journal of Endocrinology</title>
+    <title-short>Eur J Endocrinol</title-short>
+    <id>http://www.zotero.org/styles/european-journal-of-endocrinology</id>
+    <link href="http://www.zotero.org/styles/european-journal-of-endocrinology" rel="self"/>
+    <link href="http://www.zotero.org/styles/vancouver" rel="template"/>
+    <link href="http://www.eje-online.org/site/misc/For-Authors.xhtml#refs" rel="documentation"/>
+    <author>
+      <name>Sylvère Störmann</name>
+      <email>sylvere.stoermann@med.uni-muenchen.de</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0804-4643</issn>
+    <eissn>1479-683X</eissn>
+    <updated>2016-12-29T23:48:11+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name initialize-with="" name-as-sort-order="all" sort-separator=" "/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <group prefix="Eds ">
+      <names variable="editor" suffix=". ">
+        <name and="symbol" initialize-with="" name-as-sort-order="all" sort-separator=" ">
+          <name-part name="given" text-case="uppercase"/>
+        </name>
+      </names>
+    </group>
```

Comment:
that must be a typo in the specs and we don't validate this, so it's OK to allow imo. Otherwise we're in trouble: 
We have cs:label before cs:name all over the place, e.g. for pretty much any verb form of the lable, see e.g. https://github.com/citation-style-language/styles/blob/master/chicago-fullnote-bibliography.csl#L56